src/core/index_meta.rs

```diff
@@ -1,4 +1,5 @@
 use core::SegmentMeta;
+use indexer::Opstamp;
```

Comment:
I think we can put `Opstamp` in the toplevel namespace.
(ie: `tantivy::Opstamp`)